modules/joinmarket.nix

```diff
@@ -187,29 +177,36 @@ in {
         # Generating wallets (jmclient/wallet.py) is only supported for mainnet or testnet
         ExecStartPost = mkIf (bitcoind.network == "mainnet") (nix-bitcoin-services.privileged ''
           walletname=wallet.jmdat
-          pw=$(cat "${secretsDir}"/jm-wallet-password)
-          mnemonic=${secretsDir}/jm-wallet-seed
           if [[ ! -f ${cfg.dataDir}/wallets/$walletname ]]; then
-            echo Create joinmarket wallet
-            # Use bash variables so commands don't proceed on previous failures
-            # (like with pipes)
-            cd ${cfg.dataDir} && \
-              out=$(sudo -u ${cfg.user} \
-              ${nbPkgs.joinmarket}/bin/jm-genwallet \
-              --datadir=${cfg.dataDir} $walletname $pw)
-            recoveryseed=$(echo "$out" | grep 'recovery_seed')
-            echo "$recoveryseed" | cut -d ':' -f2 > $mnemonic
+            echo "Create wallet"
+            pw=$(cat "${secretsDir}"/jm-wallet-password)
+            cd ${cfg.dataDir}
+            sudo -u ${cfg.user} ${nbPkgs.joinmarket}/bin/jm-genwallet --datadir=${cfg.dataDir} $walletname $pw \
+              | grep 'recovery_seed' \
```

Comment:
Try replacing with `grep 'recoveryy_seed'` and you will see that the command succeeds eventhough the `grep` fails. That is why I initially added the comment and the bash variables.

Reply:
I've checked that before, it's failing as expected.
Try it yourself via `run-tests.sh -s joinmarket container -d --run c journalctl -u joinmarket`.

Reply:
I deployed with `deploy-qemu-vm.sh` and verified that the `joinmarket` service starts without failure but the `/secrets/jm-wallet-seed` file is empty, when testing with `recoveryy_seed`.